decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,10 +1,32 @@
 <div class="wrapper">
   <div class="row">
     <div class="large-8 large-centered">
-    <div class="static__message">
-      <h1 class="heading1 page-title"><%= t(".title") %></h1>
-      <p><%= t(".try_later") %></p>
+      <div class="static__message">
+        <h1 class="heading1 page-title"><%= t(".title") %></h1>
+        <p><%= t(".try_later") %></p>
+        <div class="table-scroll">
+          <table class="table-list">
+          <% @info_hash.map do |key, val| %>
+            <tr>
+              <th><%= t(".#{key}") %></th>
+              <td> <%= val %></td>
```

Comment:
Let's remove the unnecessary space from here:
```suggestion
              <td><%= val %></td>
```

Note that you can also commit the suggestion directly from the GitHub UI to make it faster.